test/e2e/README.md

```diff
@@ -0,0 +1,14 @@
+# Running the Tests
+1. Setup a Kubernetes Cluster and setup GCS inside it as the [GCS deployment
+   scripts recommend][1].
+2. Make sure `kubectl` points to that cluster by doing the following:
+
+```
+    KUBECONFIG=/path/to/kubeconfig
+```
```

Comment:
looks good to me (need to remove commented code)
just a question do e2e tests make use of kubectl or it will make rest call to the kubernetes?

Reply:
From what I know, it uses the `client-go` libs to make REST API calls rather than `kubectl` directly.

Reply:
means does it reads the config from the default location ($HOME/.kube/config) or from ENV variable KUBECONFIG?

Reply:
It will read from the default location.

Reply:
If it is the case that it will read from a default location, is the environment variable used at all for the tests?